apps/server/src/modules/roster/service/feathers-roster.service.ts

```diff
@@ -122,9 +122,17 @@ export class FeathersRosterService {
 
 			const room = await this.roomService.getSingleRoom(id);
 
+			const roomMembers = await this.roomMembershipService.getRoomMembershipAuthorizable(room.id);
+			const hasOwner = roomMembers.members.some((member) =>
+				member.roles.some((role) => role.name === RoleName.ROOMOWNER)
+			);
+			if (!hasOwner) {
+				throw new NotFoundLoggableException(RoomMembershipAuthorizable.name, { roomId: room.id });
```

Comment:
I'm wondering if we could use the more general `NotFoundLoggableException` for locked rooms ans courses as well?